### ISSUE TITLE
Reviews by non-native speakers should be deleted and disabled

### ISSUE THREAD

**Issue by KK-kaku** [Tatoeba/tatoeba2#3080]
**Story**
Reviews by non-native speakers are often incorrect, and there is no way to tell a native review from a non-native review on the sentence page.

*Example*
https://tatoeba.org/ja/sentences/show/3459009
This sentence is perfectly correct, but a non-native speaker has added an incorrect "Not OK" review.

**Idea**
All reviews made by non-native speakers should be deleted.
Reviewing sentences in non-native languages should be disabled by the system.

**Comment by vxern** [comment]
The example you gave is from a user with no owned sentences and no comments, who may have either mistakenly reviewed the sentence, or intentionally considered it as incorrect.

Why should one user's wrong contribution cause the whole group of non-native reviewers to be impeded?

**Comment by jiru** [comment]
I agree with @vxern. It looks like that particular review gets into the way of corpus maintainers who use this feature to keep track of sentences that need to be changed. I believe this is the actual reason this issue was opened, but maybe @KK-kaku can confirm?

I believe non-natives having a good-enough understanding of the language are also capable of flagging sentences with "not OK" or "unsure".

The problem could be solved in a way that does not prevent such contributions.

**Comment by DJ-Saidez** [comment]
I think the main issue here is that corpus maintainers don't have a way to remove outdated or faulty reviews from other users, and I feel like they should. That would resolve this issue I believe.

**Comment by ckjpn** [comment]
Perhaps the title should be changed to.

"Make it possible for Corpus Maintainers to edit reviews"

This would allow Corpus Maintainers to eliminate these kinds of errors, and als to errors that appear when Horus moves bad ratings from a corrected sentence to a pre-existing matching sentence.

**Comment by KK-kaku** [comment]
Changed the title.
